src/views/menus/AccountMenu.tsx

```diff
@@ -449,42 +450,47 @@ const AssetActions = memo(
     stringGetter: StringGetterFunction;
   }) => (
     <div tw="inlineRow">
-      {[
-        withOnboarding &&
-          complianceState === ComplianceStates.FULL_ACCESS && {
-            dialog: DialogTypes.Deposit(),
-            iconName: IconName.Deposit,
-            tooltipStringKey: STRING_KEYS.DEPOSIT,
-          },
-        withOnboarding &&
-          hasBalance && {
-            dialog: DialogTypes.Withdraw(),
-            iconName: IconName.Withdraw,
-            tooltipStringKey: STRING_KEYS.WITHDRAW,
-          },
-        hasBalance &&
-          complianceState === ComplianceStates.FULL_ACCESS && {
-            dialog: DialogTypes.Transfer({ selectedAsset: asset }),
-            iconName: IconName.Send,
-            tooltipStringKey: STRING_KEYS.TRANSFER,
-          },
-      ]
-        .filter(isTruthy)
-        .map(({ iconName, tooltipStringKey, dialog }) => (
-          <WithTooltip
-            key={tooltipStringKey}
-            tooltipString={stringGetter({ key: tooltipStringKey })}
-            tw="[--tooltip-backgroundColor:--color-layer-5]"
-          >
-            <$IconButton
-              key={dialog.type}
-              action={ButtonAction.Base}
-              shape={ButtonShape.Square}
-              iconName={iconName}
-              onClick={() => dispatch(openDialog(dialog))}
-            />
-          </WithTooltip>
-        ))}
+      {/* Need to wrap in Item to enable 'dismiss dropdown on click' functionality
+          In general, any CTA in a dropdown should be wrapped in an Item tag
+       */}
+      <Item>
```

Comment:
I feel like Item should be around each of them, not a big item wrapping all of them.

Reply:
+1 - I think we could also just rename to AssetActionsItem (since it's only used in AccountMenu)

Reply:
ooo okay i'm down

Reply:
@moo-onthelawn hmm i'd like to avoid creating another layer of indirection here but down to update if it makes it easier to read for you!